pkgs/applications/virtualization/xen/4.18/4.18.nix

```diff
@@ -0,0 +1,194 @@
+{
+  lib,
+  callPackage,
+  fetchpatch,
+  fetchgit,
+  fetchFromGitHub,
+  ocaml-ng,
+
+  withInternalQEMU ? true,
+
+  withInternalTraditionalQEMU ? true,
+
+  withInternalSeaBIOS ? true,
+  withSeaBIOS ? !withInternalSeaBIOS,
+  seabios,
+
+  withInternalOVMF ? true,
+  withOVMF ? !withInternalOVMF,
+  OVMF,
+
+  withInternalIPXE ? true,
+  withIPXE ? !withInternalIPXE,
+  ipxe,
+
+  # QEMU
+  udev,
+  pciutils,
+  xorg,
+  SDL,
+  pixman,
+  acl,
+  glusterfs,
+  spice-protocol,
+  usbredir,
+  alsa-lib,
+  glib,
+  python311,
+
+  # OVMF
+  nasm,
+  bash,
+  ...
+}@args:
+
+assert withInternalSeaBIOS -> !withSeaBIOS;
+assert withInternalOVMF -> !withOVMF;
+assert withInternalIPXE -> !withIPXE;
+
+with lib;
```

Comment:
```suggestion
```

Reply:
Removed on both `generic.nix` and `4.18.nix`.